src/components/Avatar/Avatar.scss

```diff
@@ -14,6 +14,10 @@ $large-size: rem(60px);
   border-radius: $large-size / 2;
   user-select: none;
 
+  @media (-ms-high-contrast: active) {
+    border: 1px solid ms-high-contrast-color('text');
+  }
```

Comment:
@sarahill I've finally been able to circle back to this and add a border for high contrast mode:

![image](https://screenshot.click/WinDev1912Eval_Running_2020-01-20_16-09-03.png)